FWCore/ParameterSet/python/Types.py

```diff
@@ -109,7 +109,7 @@ def dumpPython(self, options=PrintOptions()):
             v+=".untracked"
         if hasattr(self.__type, "__name__"):
             return v+'.'+self.__type.__name__
-        return v+'.'+self.__type.dumpPython(options)
+        return v+'.'+self.__type.dumpPython()
```

Comment:
Why not passing `options` anymore? Or was this causing the extra indentation that is now removed (as visible in the tests below)?

Reply:
It removes the extra indention.

Reply:
After seeing the additional failures of unit tests, I determined we do want the extra indents.